src/frontend/js/reducers/security.js

```diff
@@ -1,23 +1,41 @@
 import {
   CHECK_LOGGED_IN,
+  LOGIN_REQUESTED,
+  LOGIN_SUCCSSFUL,
+  LOGIN_FAILED,
 } from '../actions/Security';
 
 function checkLoggedIn() {
   const token = localStorage.getItem('token');
   if (token == null) {
     return { loggedIn: false, family: null };
   }
+  return userLoggedIn(token);
+}
+
+function userLoggedIn(token) {
+  localStorage.setItem('token', token);
```

Comment:
I see you extended this to support the `security` method below, but it feels weird that `setItem` is called after `getItem` is called in `checkLoggedIn`. Calling `setItem` before `userLoggedIn` might be better even if you are repeating yourself (slightly)? I don't think it matters that much, I just noticed it.

Reply:
You're right. I agree, I DRY'd it too much, didn't even think about it.